css/index.css

```diff
@@ -104,14 +137,45 @@ p {
 .middle-content .boxes .box {
     width: 12.5%;
     height: 100px;
-    background: black;
     margin: 20px 2.5%;
     color: white;
     display: flex;
     align-items: center;
     justify-content: center;
 }
 
+.box1 {
```

Comment:
good job making a new class just for the color and not rewritting the box properties  from the box class